cost/aws/idle_compute_instances/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v5.2
+
+- Fixed `tags` incident field output
+- Added `resourceName`, `lookBackPeriod`, `threshold`, `memoryThreshold` and `thresholdType` incident fields
```

Comment:
The CHANGELOG should make reference to what the user would see when using the policy, not the code itself. These should be listed with the names the user would see when looking at an incident.